os/arch/arm/src/s5j/Kconfig

```diff
@@ -358,3 +358,11 @@ config S5J_SENSOR_PPD42NS
 	default n
 
 endmenu
+
+comment "S5J PM Options"
```

Comment:
This has only one config. Do we need to make a comment only for this?

Reply:
not necessary, will remove it